pages/propose-a-project.vue

```diff
@@ -1,42 +1,45 @@
-<template>
-    <div>
-        <SmallHero />
-        <div class="container mt-4 mb-3">
-            <div class="section-content">
-                <div class="row g-gs justify-content-between">
-                    <div class="col">
-                        <div class="text-block">
-                            <h1 class="subtitle">PROPOSE A PROJECT</h1>
-                            <p>
-                                We are a non-profit organization focused on applying technology to maximize impact solving social problems.
-                                Complete the following form with the details of your project.
-                            </p>
-                            <Typeform class="w-100 mb-3" style="height: 500px" :hide-headers="true" :hide-footer="true" :opacity="0" :on-submit="onSubmit" url="https://info548263.typeform.com/to/JkDG5C" />
-                        </div>
-                    </div>                    
-                </div>
-            </div>
-        </div>
-    </div>
-</template>
-
-<script>
-import SmallHero from '@/components/smallHero'
-import Typeform from 'nuxt-typeform'
-
-export default {
-    components: {
-        SmallHero,
-    },
-    methods: {
-        onSubmit () {
-            this.$ga.event({
-                eventCategory: 'Forms',
-                eventAction: 'Send Project Form',
-                eventLabel: 'projectSignup',
-                eventValue: 1
-            })
-        }
-    }
-}
-</script>
+<template>
+  <div>
+    <SmallHero />
+    <div class="container mt-4 mb-3">
+      <div class="section-content">
+        <div class="row g-gs justify-content-between">
+          <div class="col">
+            <div class="text-block">
+              <h1 class="subtitle">PROPOSE A PROJECT</h1>
+              <p>
+                We are a non-profit organization focused on applying technology to maximize impact solving social problems.
+                Complete the following form with the details of your project.
+              </p>
+              <!-- <Typeform class="w-100 mb-3" style="height: 500px" :hide-headers="true" :hide-footer="true" :opacity="0" :on-submit="onSubmit" url="https://info548263.typeform.com/to/JkDG5C" /> -->

```

Comment:
The part about custom-form can be commented and the typeform one uncommented so the code can be pushed without affecting the current behaviour